previewer_jupyter/indico_previewer_jupyter/templates/ipynb_preview.html

```diff
@@ -1,31 +1,42 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <title>Jupyter Notebook Preview</title>
-        <style>
-            {{ css_code }}
-        </style>
-    </head>
-    <body>
-        {{ html_code | safe }}
-        <script type="text/javascript" nonce="{{ nonce }}">
-            var MathJax = {
-                tex2jax: {
-                    inlineMath: [ ['$','$'], ["\\(","\\)"] ],
-                    displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
-                    processEscapes: true,
-                    processEnvironments: true
-                },
-                // Center justify equations in code and markdown cells. Elsewhere
-                // we use CSS to left justify single line equations in code cells.
-                displayAlign: 'center',
-                "HTML-CSS": {
-                    styles: {'.MathJax_Display': {"margin": 0}},
-                    linebreaks: { automatic: true }
-                }
-            };
-        </script>
-        <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_HTML">
-        </script>
-    </body>
+  <head>
+    <title>Jupyter Notebook Preview</title>
+    <style>
+      {{ css_code }}
+    </style>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/katex@0.15.3/dist/katex.min.css"
+      integrity="sha384-KiWOvVjnN8qwAZbuQyWDIbfCLFhLXNETzBQjA/92pIowpC0d2O3nppDGQVgwd2nB"
+      crossorigin="anonymous"
+    />
+    <script
+      src="https://cdn.jsdelivr.net/npm/katex@0.15.3/dist/katex.min.js"
+      integrity="sha384-0fdwu/T/EQMsQlrHCCHoH10pkPLlKA1jL5dFyUOvB3lfeT2540/2g6YgSi2BL14p"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    {{ html_code | safe }}
+
+    <script type="module" nonce="{{ nonce }}">
+      import renderMathInElement from 'https://cdn.jsdelivr.net/npm/katex@0.15.3/dist/contrib/auto-render.mjs';
```

Comment:
can this be protected via https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity? i don't like the idea of embedding random files from a CDN with no guarantees that they are never modified.

alternatively just shipping it w/ the plugin would be an option

Reply:
Right, I guess I can add back the integrity...